cmd/temporal-mcp/main.go

```
@@ -212,6 +215,59 @@
 	return writer.String(), nil
 }
 
+// registerGetWorkflowHistoryTool registres a tool that gets workflow histories
+func registerGetWorkflowHistoryTool(server *mcp.Server, tempClient client.Client) error {
```

golangci-lint (1.64.8)

func `registerGetWorkflowHistoryTool` is unused

(unused)